### ISSUE TITLE
[BUG] Unhandled exception: Instance of 'ProcessTokenException'

### ISSUE THREAD

**Issue by mahindev-ai** [mark-nicepants/figma2flutter#25]
### :information_source: Info

**Version: `e.g. v2.8.15`**

### :speech_balloon: Description

**Enter a description of your problem here.**<br />
After following the steps in the documentation and later the steps in an "Issues" from this repository [issue](https://github.com/mark-nicepants/figma2flutter/issues/19#issue-2096613560) the following errors came out:

```
C:\Users\user\test>C:/Users/user/Documents/figma2flutter/bin/figma2flutter.exe --input ./design/tokens/ --output ./lib/tokens
Loading `$metadata`, `$themes` and design token files from ./design/tokens

Found 4 themes, generating code
Unhandledexception:
Instance of 'ProcessTokenException'
#0 Processor._safeProcess (package:figma2flutter/processor.dart:52)
#1 Processor._loopProcess (package:figma2flutter/processor.dart:42)
#2 Processor.process (package:figma2flutter/processor.dart:24)
#3 _processTokens (file:///c:/users/user/documentos/figma2flutter/bin/figma2flutter.dart:132)
#4 main (file:///c:/users/user/documentos/figma2flutter/bin/figma2flutter.dart:94)
<asynchronous suspension>
```

### :scroll: tokens file you tried to import

[Tokens](https://1drv.ms/f/s!AiT2PAPgCnZprRwBYQnfRtY2y2dz?e=N4akDa)
**It may be that a certain way of exporting or new options are not supported. When providing a json file we can hunt down problems more quickly**

**Comment by freemansoft** [comment]
Can you try pulling the latest and using it instead of the last release?  There should be more diagnostic data when a ProcessTokenException is thrown.

**Comment by freemansoft** [comment]
Closing this as fixed with 0.3.0 alpha.

Note the token file on one drive is not accessible by me
